build-logic/build_properties.gradle.kts

```diff
@@ -13,10 +13,9 @@ extra.apply {
      * CI. Read more about our versioning here:
      * https://simprints.atlassian.net/wiki/spaces/KB/pages/1761378305/Releasing+Simprints+ID
      *
-     * Dev version >= 2023.4.1 is required for receiving encryption Tokens from BFSID [CORE-2502]
-     * Dev version >= 2023.4.0 is required for receiving new fingerprint configurations  [CORE-3033]
+     * Dev version >= 2024.2.1 is required for receiving biometric sdk age restrictions
      */
-    set("VERSION_NAME", "2024.1.1")
+    set("VERSION_NAME", "2024.2.1")
```

Comment:
As per @ybourgery  the 2024.2.1-dev is the release version that can access the new config

Reply:
@luhmirin-s The version that will receive the new project configuration is for version greater than `2024.2.0`. If in dev you are using `2024.2.0` and because you are adding the suffix `-dev+1` it would lower than `2024.2.0` (i.e. 2024.2.0 > 2024.2.0-dev+1)